app/javascript/react/containers/PodcastsIndexContainer.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from "react";
+import PodcastTile from "../components/PodcastTile";
+
+const PodcastsIndexContainer = (props) => {
+  const [podcasts, setPodcasts] = useState([]);
+
+  useEffect(() => {
+    fetch("/api/v1/podcasts")
+      .then((response) => {
+        if (response.ok) {
+          return response;
+        } else {
+          let errorMessage = `${response.status} (${response.statusText})`;
+          let error = new Error(errorMessage);
+          throw error;
+        }
+      })
+      .then((response) => response.json())
+      .then((body) => {
+        setPodcasts(body);
+      })
+      .catch((error) => console.error(`Error in fetch: ${error.message}`));
+  }, []);
+
+  const podcastTiles = podcasts.map((podcast) => {
+    return <PodcastTile key={podcast.id} podcast={podcast} />;
+  });
+
+  if (podcasts.length === 0) {
```

Comment:
I would recommend moving this conditional up to where you define `podcastTiles`, so if there are no podcasts you will conditionally return your p tag text, or map over all the podcasts. That way you have just one  final return for this component as {podcastTiles}